src/test/regress/expected/multi_tenant_isolation.out

```diff
@@ -497,20 +497,26 @@ SET citus.override_table_visibility TO false;
       Schema      |            Name            | Type  |         Owner
 ---------------------------------------------------------------------
  Tenant Isolation | lineitem_streaming         | table | mx_isolation_role_ent
+ Tenant Isolation | lineitem_streaming_1230000 | table | mx_isolation_role_ent
+ Tenant Isolation | lineitem_streaming_1230010 | table | mx_isolation_role_ent
```

Comment:
would be good to avoid these changes, especially since they could be flappy

Reply:
Fixed.